cmd/otel-allocator/config/config.go

```diff
@@ -36,11 +38,14 @@ import (
 )
 
 const (
-	DefaultResyncTime                        = 5 * time.Minute
-	DefaultConfigFilePath     string         = "/conf/targetallocator.yaml"
-	DefaultCRScrapeInterval   model.Duration = model.Duration(time.Second * 30)
-	DefaultAllocationStrategy                = "consistent-hashing"
-	DefaultFilterStrategy                    = "relabel-config"
+	DefaultResyncTime                          = 5 * time.Minute
+	DefaultConfigFilePath       string         = "/conf/targetallocator.yaml"
+	DefaultCRScrapeInterval     model.Duration = model.Duration(time.Second * 30)
+	DefaultAllocationStrategy                  = "consistent-hashing"
+	DefaultFilterStrategy                      = "relabel-config"
+	DefaultHttpsCAFilePath                     = "/conf/tls/ca.crt"
+	DefaultHttpsTLSCertFilePath                = "/conf/tls/tls.crt"
+	DefaultHttpsTLSKeyFilePath                 = "/conf/tls/tls.key"
```

Comment:
I wonder if these should have default values at all. What is the benefit of having these?

Reply:
Following the style of having a default value for `DefaultConfigFilePath`.
Should these be different?

Reply:
Honestly, that one probably shouldn't have a default either. There are users of standalone target allocator out there, and I try to avoid coupling this configuration too hard to what the operator wants. For these values in particular, since HTTPS is disabled by default, I think the user should be forced to supply them.